### DISCUSSION TITLE
Can the kernel tools patch wan sfp driver

### DISCUSSION THREAD

**Discussion by rcmpayne** [fabianishere/udm-kernel-tools#39] [Q&A]
I have Bell fibre 1.5G which requires the udm sfp wan port to be set to 2.5G vs 1G or 10G. Does this allow us to patch the drivers? I have the changes needed which is only two code lines

**Comment by fabianishere** [comment]
If you need to modify your kernel for this, yes this would work. However, it might be that this is a limitation of the UniFi (or SFP+ transceiver) firmware and not the Linux kernel.
